tests/utils/test_utils.py

```diff
@@ -25,3 +25,26 @@ def test_get_dual(dict5):
     assert dual[6] == [2, 3]
     assert dual[7] == [3]
     assert dual[8] == [3]
+
+
+def test_convert_labels_to_integers(hypergraph1):
```

Comment:
Can you please add another test where the original hypergraph has nodes/edges labeled by non-consecutive integers? And/Or, a test where the original hypergraph has a single node labeled `0` but all other nodes are labeled by strings.

Reply:
This will catch the bug I point out later.

Reply:
Done.